drivers/uart_seds/work/assembler/implem/default/C/src/assembler.c

```diff
@@ -0,0 +1,156 @@
+/* Body file for function Assembler
+ * Generated by TASTE (kazoo/templates/skeletons/c-body/function.tmplt)
+ * You can edit this file, it will not be overwritten
+
+    !! IMPORTANT                                                        !!
+    !! When you modify your design (interface view), you must update    !!
+    !! the procedures corresponding to the provided interfaces in this  !!
+    !! file. The up-to-date signatures can be found in the .ads file.   !!
+*/
+#include "assembler.h"
+
+#include <assert.h>
+
+#include <FreeRTOS.h>
+#include <semphr.h>
+
+#include <Escaper.h>
+
+__attribute__((section(".sdramMemorySection")))
+static asn1SccUartHwas uart;
+
+__attribute__((section(".sdramMemorySection")))
+static uint64_t sentBytes;
+__attribute__((section(".sdramMemorySection")))
+static uint64_t bytesToSend;
+
+__attribute__((section(".sdramMemorySection")))
+static Escaper escaper;
```

Comment:
Wasn't this part a short term sollution? Shouldn't it be placed in private data?

Reply:
It should be, however at this point we decided to store all the above vars as globals inside assembler. It will be polished when we implement RS785